web/src/routes/auth/login/+page.server.ts

```diff
@@ -1,14 +1,20 @@
+import { AppRoute } from '$lib/constants';
 import { redirect } from '@sveltejs/kit';
 import type { PageServerLoad } from './$types';
 
 export const load = (async ({ locals: { api } }) => {
 	const { data } = await api.userApi.getUserCount(true);
 	if (data.userCount === 0) {
 		// Admin not registered
-		throw redirect(302, '/auth/register');
+		throw redirect(302, AppRoute.AUTH_REGISTER);
 	}
 
+	// TODO: Figure out how to get correct redirect URI server-side.
```

Comment:
It returns a URL object, but `url.origin + url.pathname` results in `http://localhost/auth/login` without port 2283. And from memory, it would always return http even when accessing over https. I think it has something to do with the proxy not passing the correct `Host` header, but I'll have to investigate further.

Reply:
Sounds good. If needed, happy to help test the OAuth portion of this once you figure it out.